apps/src/p5lab/spritelab/commands/eventCommands.js

```diff
@@ -17,6 +17,10 @@ export const commands = {
     this.addEvent('collectData', {}, callback);
   },
 
+  everySeconds(n, unit, callback) {
+    this.addEvent('everySeconds', {n, unit}, callback);
```

Comment:
Is it strange to call this `everySeconds` when the unit isn't always "seconds"? We also have an `atTime` event that works with frames as well as seconds. Maybe this could be `atTimeInterval`?